crates/stc_ts_type_checker/tests/conformance.pass.txt

```diff
@@ -1,2500 +1,5001 @@
 Symbols/ES5SymbolProperty1.ts
+Symbols/ES5SymbolProperty1.ts
```

Comment:
why duplicate conformance...
I don't know this reason

I use scripts/check.sh, scripts/test.sh, cargo test...

Reply:
You have to wait for `./scripts/check.sh` to finish